domain/schema_validator.go

```diff
@@ -164,19 +164,18 @@ func (s *schemaValidator) Update(leaseGrantTS uint64, oldVer, currVer int64, cha
 // isRelatedTablesChanged returns the result whether relatedTableIDs is changed
 // from usedVer to the latest schema version.
 // NOTE, this function should be called under lock!
-func (s *schemaValidator) isRelatedTablesChanged(currVer int64, tableIDs []int64) (transaction.RelatedSchemaChange, bool) {
-	res := transaction.RelatedSchemaChange{}
+func (s *schemaValidator) isRelatedTablesChanged(currVer int64, tableIDs []int64) bool {
```

Comment:
Maybe we could add some comments to demonstrate that the schema validator now is only used when MDL is turned off and MDL would be enabled by default.

Reply:
We should refine the comment later